pkgs/by-name/op/opensnitch/go.mod

```diff
@@ -1,19 +1,19 @@
 module github.com/evilsocket/opensnitch/daemon
```

Comment:
Why do we need to vendor this file? it exists uostream https://github.com/evilsocket/opensnitch/blob/master/daemon/go.mod

Reply:
Without it I get this error:
```
       > go: downloading github.com/BurntSushi/toml v0.4.1
       > go: downloading golang.org/x/xerrors v0.0.0-20200804184101-5ec99f83aff1
       > go: downloading golang.org/x/mod v0.5.1
       > go: github.com/evilsocket/opensnitch/daemon/firewall/nftables/nftest: package github.com/evilsocket/opensnitch/daemon/firewall/nftables/nftest imports github.com/vishvananda/netns from implicitly required module; to add missing requirements, run:
       >       go get github.com/vishvananda/netns@v0.0.0-20200728191858-db3c7e526aae
```